.github/workflows/approve-snapshots.yml

```diff
@@ -55,6 +55,7 @@ jobs:
       - name: Run Cypress 🌳
         run: pnpm cypress:run --env updateSnapshots=true
         env:
+          CYPRESS_BASE_URL: http://localhost:3000
```

Comment:
Note we may instead chose to `serve` the app at port `5173` to have a set-up closer to what we had before